crates/sargon/src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/success_response/pre_authorization/subintent_response.rs

```diff
@@ -0,0 +1,60 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct WalletToDappInteractionSubintentResponseItem {
+    /// A hex encoded signed partial transaction.
+    #[serde(rename = "signedPartialTransaction")]
+    pub encoded_signed_partial_transaction: String,
```

Comment:
prefer keeping the `SignedSubintent` intact.

And then you can use `#[serde_as]` together with `#[serde_as(as = `, see https://github.com/radixdlt/sargon/blob/4a5a557b3e373421cf0ed2c26d2c2db51a6d85f0/crates/sargon/src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/success_response/auth/auth_proof.rs#L3-L7

more info https://docs.rs/serde_with/latest/serde_with/